src/components/common/charts/styles/chartStyles.css

```
@@ -717,7 +705,7 @@
 .chart-dot:hover {
   transform: scale(1.3);
   stroke-width: 2;
-  filter: drop-shadow(0 0 4px var(--accent-primary));
+  filter: drop_shadow(0 0 4px var(--accent-primary));
```

Biome (1.9.4)

[error] Unexpected unknown function: drop_shadow

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)